graylog2-web-interface/src/views/components/widgets/EditWidgetFrame.tsx

```diff
@@ -71,6 +75,8 @@ const onSubmit = (values, widget: Widget) => {
 };
 
 const EditWidgetFrame = ({ children }: Props) => {
+  const config: SearchesConfig = useStore(SearchConfigStore, ({ searchesClusterConfig }) => searchesClusterConfig);
```

Comment:
IMO we can remove the explicit `SearchesConfig` type. It should be inferred from `SearchConfigStore`'s type.

Reply:
@dennisoelkers Updated!